### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by arun-gopal**
Integration with Trello API

Screenshot from 2021-05-09 16-10-54

### CODE REVIEW

File: README.md
Comment (JackMeadDev) [inline README.md:39]: Nice docs, but it would be good to also add these to your .env.template file so that a user has a natural prompt there as well

File: todo_app/trello_client.py
Comment (JackMeadDev) [inline todo_app/trello_client.py]: I can't see any evidence that you use this function (or the one it calls) - let's clear it out in that case, we can always recover it from git if we want it later

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py:4]: It looks like you no longer user `session_items` at all - we should get rid of that now in that case (and if that's correct, we could also get rid of the config/secret key as we no longer need those)

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py:11]: Here you're still directly passing through the responses json to the template - this doesn't give us much control over the items we're passing to the frontend, and means they are closely tied to Trello. Instead, we should move to converting Trello's responses into a class under our control, and using that elsewhere in our code.

**Review by JackMeadDev — CHANGES_REQUESTED** [review]
This looks good, I was able to easily follow your instructions and view/create/move todos on the site backed by Trello. I've left a few thoughts below, but importantly you've not done Part 5 here - creating a class for to-do items - so we should do that before this can be approved. Feel free to get in touch if you have any questions regarding that

File: README.md
Comment (arun-gopal) [reply README.md:39]: Done, thanks.

File: todo_app/app.py
Comment (arun-gopal) [reply todo_app/app.py:4]: Done, thanks.

File: todo_app/trello_client.py
Comment (arun-gopal) [reply todo_app/trello_client.py]: Removed it, thanks.

**Comment by arun-gopal** [comment]
> This looks good, I was able to easily follow your instructions and view/create/move todos on the site backed by Trello. I've left a few thoughts below, but importantly you've not done Part 5 here - creating a class for to-do items - so we should do that before this can be approved. Feel free to get in touch if you have any questions regarding that

This has been completed now, thanks.

File: todo_app/app.py
Comment (arun-gopal) [reply todo_app/app.py:11]: This has been completed as suggested. Thanks.

**Review by Jonesey13 — APPROVED** [review]
Looks good to me! Free free to merge this PR in when you are ready 👍

**Pull request merged**
